registrations/tasks.py

```diff
@@ -43,3 +44,41 @@ def send_registration_to_openhim(
     )
     response.raise_for_status()
     return (response.status_code, response.headers, response.content)
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def send_registration_to_rapidpro(
+    contact, msisdn, referral_msisdn, channel, clinic_code, timestamp
+):
+    # Create/Update contact
+    group_name = "nurseconnect-%s" % channel.lower()
+    group = tembaclient.get_groups(name=group_name).first()
+    contact_data = {
+        "preferred_channel": channel.lower(),
+        "registered_by": referral_msisdn,
```

Comment:
I think we want to do the same thing here as when we submit to Jembi, we want to set it to the user's cellphone number if they registered themselves, ie `"registered_by": referral_msisdn or msisdn`